docs/migration-guides/1.5-1.6.md

```diff
@@ -0,0 +1,13 @@
+# Migrate from v1.5 to v1.6
+
+>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.
+
+## Service Management
+
+We have migrated the Service Catalog to the CRDs in this [PR](https://github.com/kubernetes-sigs/service-catalog/pull/2630). To support upgrade from earlier versions, we created a migration job which migrates their Service Catalog related resources on upgrade.
+
+In order to make a successful upgrade of Kyma, the Service Catalog resources can't be in the unfinished/deleted state. Otherwise our upgrade job can fail.
```

Comment:
```suggestion
In order to perform the successful Kyma upgrade, the Service Catalog resources can't be in the unfinished or deleted state. Otherwise, the upgrade job can fail.
```

Reply:
I miss the information here. What the user has to do exactly?